src/JobBoard/JobBoard.js

```diff
@@ -52,254 +67,115 @@ class JobBoard extends Component {
     getActiveUser(userKeys) {
         let value = this.context;
         for (var i = 0; i < userKeys.length; i++) {
-          var curr = this.state.allUsers[userKeys[i]];
-          if (curr.username == value["username"]) {
-            activeUser.username = curr.username;
-            activeUser.trustedUsers = curr.trustedUsers;
-          }
-        }
-    }
-
-    sortJobsByDate(jsonObjects, prop, direction) {
-        var objArray = [];
-        var keys = Object.keys(jsonObjects);
-        for (var i = 0; i < keys.length; i++) {
-            var k = keys[i];
-
-            var job = {
-            };
-        }
-        var direct = arguments.length > 2 ? arguments[2] : 1;
-        if (objArray) {
-            objArray.sort(function(a,b) {
-                if(a[prop] && b[prop]) {
-                    var aDate = new Date(a[prop]);
-                    var bDatte = new Date(b[prop]);
-                }
-                return ( (a < b) ? -1*direction : ((a > b) ? 1*direction : 0) );
-            });
+            var curr = this.state.allUsers[userKeys[i]];
+            if (curr.username == value["username"]) {
+                activeUser.username = curr.username;
+                activeUser.trustedUsers = curr.trustedUsers;
+            }
         }
     }
 
-    FlatListItemSeparator = () => {
-        return (
-            <View
-                style={{
-                    height: 1,
-                    width: "100%",
-                    backgroundColor: "#607D8B",
-                }}
-            />
-        );
-    }
+//    sortJobsByDate(jsonObjects, prop, direction) {
+//        var objArray = [];
+//        var keys = Object.keys(jsonObjects);
+//        for (var i = 0; i < keys.length; i++) {
+//            var k = keys[i];
+//
+//            var job = {
+//            };
+//        }
+//        var direct = arguments.length > 2 ? arguments[2] : 1;
+//        if (objArray) {
+//            objArray.sort(function(a,b) {
+//                if(a[prop] && b[prop]) {
+//                    var aDate = new Date(a[prop]);
+//                    var bDatte = new Date(b[prop]);
+//                }
+//                return ( (a < b) ? -1*direction : ((a > b) ? 1*direction : 0) );
+//            });
+//        }
+//    }
 
-    JobItem (props) {
-        let startJSONdate = new Date(props.dataPoint.startDateTime);
-        let endJSONdate = new Date(props.dataPoint.endDateTime);
-        let startClockTime = formatTime(startJSONdate);
-        let endClockTime = formatTime(endJSONdate);
-        var trusted = false;
-        for(var i = 0; i < activeUser.trustedUsers.length; i++) {
-          if (activeUser.trustedUsers[i] == props.dataPoint.requestor) {
-            trusted = true;
-          }
-        }
-        if (props.dataPoint.volunteer == "" && monthNames[startJSONdate.getMonth()] == props.month
```

Comment:
Why did we get rid of this if statement? Jobs should only be shown on the job board if it matches the requestor filter (trusted or not) and only if the job has not already been assigned a volunteer (i.e. the volunteer attribute of the job is an empty string ""). The way I see how it's implemented now, all jobs will always be displayed on the job board. This functionality existed in the JobBoard.js code before the merge and it still needs to be here after.

Reply:
I removed this because JobItem was no longer being reference based on the changes that Catherine made on her branch (The branch that I am submitting a PR for now is made off of hers).